pages/advanced-algorithms/available-algorithms/merge.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: merge
+description: Find out how to merge nodes and relationships within the graph structure with more precision and flexibility.
+---
+
+import { Callout } from 'nextra/components'
+import { Card, Cards } from 'nextra/components'
+import GitHub from '/components/icons/GitHub'
+
+# merge
+
+The `merge` module provides the capabilities of the `MERGE` Cypher command, merging or creating nodes and relationships as per specified conditions.
+It ensures precision and coherence in managing interconnected data structures.
+
+<Cards>
+  <Card
+    icon={<GitHub />}
+    title="Source code"
+    href="https://github.com/memgraph/mage/tree/main/cpp/merge_module"
+  />
+</Cards>
+
+| Trait               | Value               |
+| ------------------- | ------------------- |
+| **Module type**     | util                |
+| **Implementation**  | C++                 |
+| **Graph direction** | directed/undirected |
+| **Edge weights**    | weighted/unweighted |
+| **Parallelism**     | sequential          |
+
+## Procedures
+
+<Callout type="info">
+You can execute this algorithm on [graph projections, subgraphs or portions of the graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph).
+</Callout>
+
+### `node()`
+
+The function provides a more flexible way of merging nodes than Cypher’s `MERGE` clause, by
+allowing labels and properties to be extracted as results of other procedures
+and sent as a list or map.
+
+{<h4> Input: </h4>}
+
+- `labels: List[string]` ➡ A list of all labels that need to be added to the new node.
+- `identProps: Map` ➡ A map with key-value pairs used for identifying which nodes will be merged.
+- `createProps: Map` ➡ A map with key-value pairs which represent properties of the node, when the procedure creates a new node.
+Equivalent to using `ON CREATE SET` for properties when using `MERGE`.
+- `mergeProps: Map` ➡ A map with key-value pairs which represent properties of the node, when the procedure merges onto a node.
+Equivalent to using `ON MATCH SET` for properties when using `MERGE`.
+
+{<h4> Output: </h4>}
+
+- `node: Node` ➡ The node which was created or merged in the procedure.
+
+{<h4> Usage: </h4>}
+
+Consider a graph with two `Human`, and one `Alien` node.
+
+```cypher
+CREATE (h:Human {name:"Mark", age: 20}), (h2:Human {name:"Eva", age: 30}), (a:Alien {name: "Gorlock", age: 20});
+```
+Use the `merge.node` procedure on all nodes, and add `galaxy` property on the existing nodes. By leaving `labels` and `identProp` empty,
```

Comment:
```suggestion
Use the `merge.node()` procedure on all nodes, and add `galaxy` property on the existing nodes. By leaving `labels` and `identProp` empty,
```

Should we add () after procedures like I did here?

Reply:
Maybe we should, yeah